app/models/lettings_log.rb

```diff
@@ -427,15 +429,17 @@ def scheme_owning_organisation_name
   end
 
   delegate :postcode, :name, :units, :type_of_unit, :mobility_type, :startdate, prefix: "location", to: :location, allow_nil: true
+  delegate :postcode_before_type_cast, :name_before_type_cast, :units_before_type_cast, :type_of_unit_before_type_cast, :mobility_type_before_type_cast, :startdate_before_type_cast, prefix: "location", to: :location, allow_nil: true
   delegate :location_admin_district, to: :location, allow_nil: true
+  delegate :location_admin_district_before_type_cast, to: :location, allow_nil: true
 
   # This is not the location_code in the db, location.id is just called code in the UI
   def location_code
     location&.id
   end
 
-  def self.to_csv(user = nil)
-    Csv::LettingsLogCsvService.new(user).to_csv
+  def self.to_csv(user = nil, is_codes_only_export:)
```

Comment:
would you not default this to `false` to prevent breaking the existing API?

Reply:
I'm happy to add it here as well. This method is only called from within the EmailCsvJob as far as I can see and the parameter defaults to false there and is passed straight through. Let me know if you think it's best to add it here as well and if so why as it doesn't seem obvious to me I would want more default values.
For interest I would also be interested to discuss and understand ruby best practices with regards something in the EmailCsvJob. I will write something on that.